src/lib.rs

```diff
@@ -89,8 +89,12 @@
 //! # Ok(())
 //! # }
 //! ```
+//!
+//! # Features
+#![doc = document_features::document_features!()]
 #![deny(missing_docs)]
 #![deny(rustdoc::broken_intra_doc_links)]
+#![cfg_attr(quicrpc_docsrs, feature(doc_cfg))]
```

Comment:
if you use the `auto` thing @flub found, you can simplify this: https://github.com/n0-computer/iroh/pull/3029

Reply:
Yeah, will do at some point. But I am afraid that I will spend another few hours on this, so I will wait for 1 iroh release before converting to the auto stuff...